shoot/diff_suppress_funcs.go

```diff
@@ -0,0 +1,23 @@
+package shoot
```

Comment:
Please do not call a file `*_funcs.go`, the name does not need to say what programming concepts it contains.
It can be called just `supress.go`, since it is the only thing it does.

Reply:
renamed to supress.go